src/3id/index.js

```diff
@@ -54,6 +64,54 @@ class ThreeId {
     })
   }
 
+  async _initDID (muportIpfs) {
+    const muportPromise = this._initMuport(muportIpfs)
```

Comment:
Not quite following how both muport and 3id are being used now, so both DIDs are created? and muport is just used for verified, what are the other back compat issues again?

Reply:
The rootStore uses the hash of the muport-did for it's name (which was a really bad decision). It also makes sure that the muport is pinned at infura.

Reply:
It's not going to be super simple to migrate fully, but we'll get there in time!